lib/rubocop/cop/style/file_read.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Favor `File.(bin)read` convenience methods.
+      #
+      # @example text mode
```

Comment:
`@example` is used for user configurable options such as `EnforcedStyle`. Annotations should be expressed using comments. The following is an example.

```suggestion
      # @example
      #
      # Examples for text mode.
```